opencensus/tags/tag_map.py

```diff
@@ -0,0 +1,30 @@
+from opencensus.tags import tag
+
+class TagMap(object):
+
+    def __init__(self, tags=None):
+        self.tags = dict(tags or {})
+        self.map = {}
+        for tag.Tag in self.tags:
```

Comment:
Should be `for tag in self.tags`, `tag.Tag` would be a `Tag` class from the `opencensus.tags.tag` module instead of the instance from the `self.tags` dict.